### PR TITLE
feat: Correct the README.md file of the project

### LINKED ISSUE #387: Add gitter channel url in readme
Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #387

### CODE REVIEW

**Comment by dreadpool2** [comment]
@iamareebjamal Should I add all the necessary files like how the app is to be setup too?

**Comment by dreadpool2** [comment]
PR Updated :+1: Please let me know if any other changes are required :+1:

File: README.md [hunk 1]
```diff
@@ -18,36 +16,40 @@ Planned features & enhancements are:
...
+A native Android app using Kotlin for writing code.
 
 ### Android App Development Set up
 
-Please find info about the set up of the Android app in your development environment [here](docs/Android_App_Setup.md).
+Please find info about the set up of the Android app in your development environment.
+
+### Sample App
+
+A sample app for testing purposes is automatically built after commits in the repository. Please download and test the app that is using the code from the dev branch [here](https://github.com/fossasia/open-event-android/blob/apk/app-release-unsigned.apk) and the code from the master branch [here](https://github.com/fossasia/open-event-android/blob/apk/open-event-master-app-release-unsigned.apk).
```
Comment (nikit19) [inline README.md]: The information is already there about the apk branch.add this info there.
Don't use specific URLs, these might change just add the link to the apk branch

Comment (dreadpool2) [reply README.md]: Added apk branch link for now. PR Updated

File: README.md [hunk 2]
```diff
@@ -76,7 +74,7 @@ We have the following branches
 	 - _versionCode_ : **Integer** : To be monotonically incremented with each merge. Failure to do so will lead to 				publishing error, and thus is a crucial step before any merge
 	 - _versionName_ : **String** : User visible version of the app. To be changed following [semantic versioning](http://semver.org/)
  * **apk** This branch contains two apk's, that are automatically generated on the merged pull request a) debug apk and b) release apk.
-
+    - Please download and test the app that is using the code from the dev branch [here](https://github.com/fossasia/open-event-android/tree/apk) and the code from the master branch [here](https://github.com/fossasia/open-event-android/tree/apk).
```
Comment (nikit19) [inline README.md]: just add one link. you are showing the same link for dev and master

File: README.md [hunk 3]
```diff
@@ -18,36 +16,36 @@ Planned features & enhancements are:
...
 
 ## Development
 
-A native Android app using  Java  for writing code. 
+A native Android app using Kotlin for writing code.
 
 ### Android App Development Set up
 
-Please find info about the set up of the Android app in your development environment [here](docs/Android_App_Setup.md).
+Please find info about the set up of the Android app in your development environment.
```
Comment (iamareebjamal) [inline README.md]: Where? Remove this section

File: README.md [hunk 4]
```diff
@@ -98,8 +96,8 @@ If you are a tester and want to test the app, you have two ways to do that:
 
 ## Open Event Android Suggestions
 
-Suggestion form link: [Form](https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform)
-Suggestion responses link: [Sheet](https://docs.google.com/spreadsheets/d/1SzR75MBEVrTY1sDM3KAMm9wltiulDAp0QT5hv9eJkKM/edit#gid=1676755229)
+- Suggestion form link: [Form](https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform)
+- Suggestion responses link: [Sheet](https://docs.google.com/spreadsheets/d/1SzR75MBEVrTY1sDM3KAMm9wltiulDAp0QT5hv9eJkKM/edit#gid=1676755229)
 
 ## License
```
Comment (iamareebjamal) [inline README.md]: Change absolute link to relative for license

Comment (dreadpool2) [reply README.md]: PR Updated

**Pull request merged**